grails-plugin/build.gradle

```diff
@@ -53,3 +53,10 @@ dependencies {
 }
 
 groovydoc.classpath += configurations.documentation
+
+tasks.named('bootJar') {
```

Comment:
Since these are grails plugins, is this something the gradle plugin can do to prevent so much duplication across all of these projects?